SparseGrids/tasgridExternalTests.cpp

```diff
@@ -933,8 +936,8 @@ bool ExternalTester::testAllWavelet() const{
         //cout << setw(wfirst) << "Rules" << setw(wsecond) << TasGrid::OneDimensionalMeta::getIORuleString(rule_wavelet) << setw(wthird) << "Pass" << endl;
         cout << setw(wfirst) << "Rules" << setw(wsecond) << "wavelet" << setw(wthird) << "Pass" << endl;
     }else{
-        cout << setw(wfirst) << "Rule" << setw(wsecond) << TasGrid::OneDimensionalMeta::getIORuleString(rule_wavelet) << setw(wthird) << "FAIL" << endl; pass = false;
-    }{ TasGrid::TasmanianSparseGrid grid;
+        cout << setw(wfirst) << "Rules" << setw(wsecond) << TasGrid::OneDimensionalMeta::getIORuleString(rule_wavelet) << setw(wthird) << "FAIL" << endl; pass = false;
```

Comment:
It was a small thing I noticed. The line in the success part of the `if` statement has `Rules`. Based on the tests for global, sequence, etc., it looked as though when the tests for all rules of a given grid class pass, the program prints `Rules so-and-so Pass` or `Rules so-and-so FAIL`. I'll undo it in this branch since it's completely unrelated to the Fourier stuff.

Reply:
* Global and Sequence relate to multiple `Rules`, hence each rule has a `Rule` pass/fail and the group has a group `Rules` pass/fail
* Wavelet rules have two variants, even though it is technically the same name
* Fourier has not variants (yet), but keep it the same as the rest